openapi/openapi/src/main/scala/endpoints/openapi/JsonSchemas.scala

```diff
@@ -71,13 +71,13 @@ trait JsonSchemas extends endpoints.algebra.JsonSchemas {
 
   lazy val stringJsonSchema: DocumentedJsonSchema = Primitive("string")
 
-  lazy val intJsonSchema: DocumentedJsonSchema = Primitive("integer")
+  lazy val intJsonSchema: DocumentedJsonSchema = Primitive("integer", format = Some("int32"))
 
-  lazy val longJsonSchema: DocumentedJsonSchema = Primitive("integer")
+  lazy val longJsonSchema: DocumentedJsonSchema = Primitive("integer", format = Some("int64"))
 
   lazy val bigdecimalJsonSchema: DocumentedJsonSchema = Primitive("number")
 
-  lazy val doubleJsonSchema: DocumentedJsonSchema = Primitive("number")
+  lazy val doubleJsonSchema: DocumentedJsonSchema = Primitive("number", format = Some("double"))
 
```

Comment:
What do you think about adding `floatJsonSchema`, following the spec?
https://swagger.io/docs/specification/data-models/data-types/#numbers

Reply:
That must be harmless, I'll add it. Thanks for pointing out!